test/unit/constructor.js

```diff
@@ -2,16 +2,16 @@ import { define } from '../../src/index';
 import helperElement from '../lib/element';
 import resolved from '../lib/resolved';
 
-describe('constructor', function () {
-  var id;
+describe('constructor', () => {
```

Comment:
The linter should in-fact look at the function and see if it is using this. If it is, then it doesn't make assumptions about which function form you should use.